vg/vgsvg/vgsvg.go

```diff
@@ -9,12 +9,13 @@ package vgsvg
 import (
 	"bufio"
 	"bytes"
-	"code.google.com/p/plotinum/vg"
 	"fmt"
-	svgo "github.com/ajstarks/svgo"
 	"image/color"
 	"io"
 	"math"
+
+	svgo "github.com/ajstarks/svgo"
```

Comment:
I'm not so sure about this change. It might hurt readability since there is a struct field named svg. I'll defer it to the original author of this code, which looks like @eaburns